cli/flox/src/commands/environment.rs

```diff
@@ -516,10 +517,29 @@ pub struct Install {
     #[bpaf(external(environment_select), fallback(Default::default()))]
     environment: EnvironmentSelect,
 
-    #[bpaf(positional("PACKAGES"), some("Must specify at least one package"))]
```

Comment:
If we remove the `some` bit we might want to manually check we have at least one package listed (if that is an actual error case)

Reply:
Good catch. I removed this because it requires you to specify at least one package _this way_, when the user could have provided at least one package with the `-i` flag. I just forgot to add a check for at least one package in total.

Reply:
Yeah i thought that's why.

Alternatively we could wrap it in an 

```
enum InstallTarget {
	Pkg(#[bpaf(argument("package"))] String),
	PkgWithId(#[bpaf(external(pkg_with_id_otion)] PkgWithIdOption))
}
```

and include that with `some`